examples/todo/js/components/TodoList.js

```diff
@@ -44,23 +44,36 @@ class TodoList extends React.Component {
 }
 
 export default Relay.createContainer(TodoList, {
+  initialVariables: {
+    status: null,
+  },
+
+  prepareVariables({status}) {
+    let nextStatus;
```

Comment:
As these files use `var` throughout, we should probably use it here too for consistency.

Separately (beyond the scope of this commit), we should switch to `const`/`let`. We might build a codemod to do most of the conversion for us automatically.